apps/web/src/lib/api/external/laskugeneraattori/actions.ts

```diff
@@ -67,6 +67,10 @@ export async function SaveAction(
 
   const attachments = formData.getAll("attachments") as File[];
 
+  const FormattedBankAccountNumber = (
```

Comment:
Nitpick: PascalCase is an unusual variable naming in JavaScript and especially this project